packages/react-reconciler/src/ReactFiberContext.old.js

```diff
@@ -316,6 +316,7 @@ function findCurrentUnmaskedContext(fiber: Fiber): Object {
           break;
         }
       }
+      // $FlowFixMe[incompatible-type] we bail out when we get a null
       node = node.return;
```

Comment:
What was the behavior before? It looks like the FlowFixMe would now hide assignments from something that's not `Fiber` or `null`. But maybe that wasn't caught before either?

Reply:
I think generally a variable seems to have a more "fixed" type now. We might want to dig into the cases separetly, but I think the `do { ... } while (node != null);` style isn't very easy to type right.